Tests/SimpleTest.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace NURFG.Tests
 {
-    public class SimpleTest
+    public partial class SimpleTest
```

Comment:
Why was this class marked `partial`?  I thought only Node classes needed to be `partial`.

Reply:
Whoops, you are absolutely right. Godot automatic port tool thought it was needed, my bad.

Think that anything that inherits `GodotObject` needs to be partial, and not just `Node` (that also inherits from `GodotObject`).